src/elements/content-sidebar/activity-feed/comment/Comment.js

```diff
@@ -139,6 +139,18 @@ class Comment extends React.Component<Props, State> {
                                 name={createdByUser.name}
                                 getUserProfileUrl={getUserProfileUrl}
                             />
+
+                            {onEdit && canEdit && !isPending ? <InlineEdit id={id} toEdit={toEdit} /> : null}
```

Comment:
does this need to be a ternary or can you just use ```... && <InlineEdit```

Reply:
I just moved the element around in the page, haven't looked at what's inside at all.